### DISCUSSION TITLE
cant able to run unit test cases

### DISCUSSION THREAD

**Discussion by DineshGon** [fastapi/fastapi#6369] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi.testclient import TestClient
import json

import os
import sys

from ..main import app
client = TestClient(app)

def test_create_vaccine():
    data = {"startTime": '3:30', 'endTime':'5:40'}
    response = client.post('/vaccine',json.dumps(data))
    assert response.status_code == 201
    assert response.json()['startTime'] == '3:30'
```

### Description

I am on the tests folder and ran the test cases by using this cmd(**python3 testvaccine.py**)
It throws the below error.

Please let me know if i am missing anything here.

```
Traceback (most recent call last):
  File "testvaccine.py", line 9, in <module>
    from ..main import app
ImportError: attempted relative import with no known parent package
```

### Operating System

macOS

### FastAPI Version

0.75.1

### Python Version

3.8.2

**Comment by DineshGon** [comment]
Please find my github repo for reference : https://github.com/fargonmer/FastAPI

**Comment by n8sty** [comment]
This isn't actually a FastAPI issue - questions like this are better suited to Stackoverflow. In general, I'd suggest learning about Python's import system. Here's some basic beginning material: https://docs.python.org/3/reference/import.html and https://realpython.com/python-import/.

**Comment by valentin994** [comment]
As n8sty suggested, just a quick tip after glancing through the test file. Avoid using relative imports.

**Comment by DineshGon** [comment]
@valentin994 
As suggested used absolute imports. But when to run the test cases facing these below issue
Traceback (most recent call last):
  File "/Users/din/Desktop/fastapi/vaccine/tests/testvaccine.py", line 1, in <module>
    **from fastapi.testclient import TestClient
ImportError: No module named fastapi.testclient**

**Comment by valentin994** [comment]
Are you using a virtual env? Are you sure it was activated?

**Comment by DineshGon** [comment]
The issue was resolved after using the Pycharm instead of VS code. 
 I can see Venv in Pycharm but not in VS code. 
PS: I appreciate if anyone can explain it in a simpler way.

**Comment by valentin994** [comment]
Pycharm activates the virtual env on startup, while in vsc you have to do it manually. The folder can be visible on both, depending on your platform (Win/Mac/Linux) you have to enable hidden folders/files to be visible. All that being said this isn't really a FastAPI issue/ As a comment before suggests look into the basic concepts first before working on an API. Look into virtual environments, and how they work. Basically it would be worthwhile to try to get a better grasp at basic python concepts before doing any API work.
